### DISCUSSION TITLE
What happens if the destination HDD is full?

### DISCUSSION THREAD

**Discussion by zpinto123** [swar/Swar-Chia-Plot-Manager#447] [Q&A]
So, I've just ran into the scenario of having 4 plots at 100% for over 11h30, when they usually take only 10h.
What I've noticed is that I have 3 destination HDDs and 1 of them is completely full now but it still shows 9 as the number of plots that are going to be written to that HDD. What's up with that? Doesn't it detect that an HDD is full and moves on to the next on the List? 
![plots stuck](https://user-images.githubusercontent.com/39332844/118412136-3d9a4b80-b690-11eb-9dad-25fcf7b78dc5.PNG)

**Comment by Kali123411** [comment]
Yeah it doesn't automatically switch to the next drive so you will have to manually copy the completed tmp files over to your new drive and then change the file extension to .plot

**Reply by swar** [reply]
Correct, There's a fix for this in the development branch.
